DESCRIPTION

```diff
@@ -17,7 +17,6 @@ Depends:
 Imports:
     DBI (>= 0.3.0),
     httr (>= 0.6),
-    RCurl,
```

Comment:
Absolutely. I have added the import as you requested. Can you please take another look?